app/src/main/java/com/ntg/user/mvpsample/addedittask/AddEditTaskFragment.java

```diff
@@ -0,0 +1,153 @@
+package com.ntg.user.mvpsample.addedittask;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.ntg.user.mvpsample.R;
+import com.ntg.user.mvpsample.data.Task;
+import com.ntg.user.mvpsample.data.source.remote.ApiClient;
+import com.ntg.user.mvpsample.data.source.remote.ApiService;
+import com.ntg.user.mvpsample.poc.PocActivity;
+
+import java.util.UUID;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+import static android.content.ContentValues.TAG;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Created by mohamed yassin on 1/28/2018.
+ */
+
+public class AddEditTaskFragment   extends Fragment implements AddEditTaskContract.View{
+    public static final String ARGUMENT_EDIT_TASK_ID = "EDIT_TASK_ID";
+
+    private AddEditTaskContract.Presenter mPresenter;
+    Task task ;
+
+    private TextView mTitle;
+
+    private TextView mDescription;
+    private ApiService mAPIService;
+
+    public static AddEditTaskFragment newInstance() {
+        return new AddEditTaskFragment();
+    }
+
+    public AddEditTaskFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        mPresenter.start();
+    }
+
+    @Override
+    public void setPresenter(@NonNull AddEditTaskContract.Presenter presenter) {
+        mPresenter = checkNotNull(presenter);
+    }
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        mAPIService= ApiClient.getClient().create(ApiService.class);
+        FloatingActionButton fab =
+                (FloatingActionButton) getActivity().findViewById(R.id.fab_edit_task_done);
+        fab.setImageResource(R.drawable.ic_done);
+        fab.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                String title = mTitle.getText().toString().trim();
+                String body = mDescription.getText().toString().trim();
+
+                    sendTask(title,body) ;
+                Intent PocIntent=new Intent(getContext(), PocActivity.class);
+                startActivity(PocIntent);
+
+            }
+        });
+
+
+    }
+
+    private void sendTask(String title,String Description) {
```

Comment:
plz remove extra lines , only one space after each method
rename method to saveTask